src/main/java/usecases/CourseEnrollmentUseCase/CourseEnrolmentInteractor.java

```diff
@@ -43,11 +40,7 @@ public EnrolmentResponseModel enrol(EnrolmentRequestModel requestModel) {
             return enrolmentPresenter.prepareFailView("You have not completed the prerequisites " +
                         "for the course");
         }
-        // deleted student id
-        sessionDbGateway.saveCourse(dbResponseModel.getCourseId(), dbResponseModel.getQuestions()); //
-        // sessionDbGateway.saveCourse(requestModel.getStudentId(), requestModel.getCourseId()); //saves course NOT with
-        // student in Session database
-        //beginCourse.runCourse(); //calls to run course
```

Comment:
I'm not sure if we're switching to henry's screen in your screen so I don't think you exactly need this but I am be wrong